index.html

```diff
@@ -1,11 +1,87 @@
 <!DOCTYPE html>
 <html lang="pt-BR">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Finbits</title>
-</head>
-<body>
- Em breve 
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="A vida é muito curta para ser burocrática. A Finbits pode reduzir em até 70% o tempo gasto com a gestão das finanças da sua empresa."/>
+    <link rel="icon" href="./images/favicon.png"/>
+
+    <title>Finbits</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:ital,wght@0,500;0,600;0,700;0,800;0,900;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/styles.css">
+  </head>
+  <body>
+    <header>
+      <a href="https://www.finbits.com.br">
+        <img src="./images/logo.png" alt="Finbits logo" class="logo"/>
+      </a>
+    </header>
+
+    <section class="introduction">
+      <div class="slogan">
+        <h1> A vida é muito curta para ser burocrática</h1>
+        <p>Reduza em até 70% o tempo gasto com a gestão das finanças da sua empresa</p>
+      </div>
+
+      <div>
+        <img src="./images/illustration_001.png" alt="Ilustração de uma plataforma web" class="illustration"/>
+      </div>
+    </section>
+
+    <section class="features">
+      <div>
+        <div>
+          <img src="./images/feature.svg" alt="Feature icone">
+        </div>
+        <div class="feature-info">
+          <h4>Cobranças e Pagamentos</h4>
```

Comment:
Para SEO seria melhor usar o h2 aqui e ajustar o style pelo CSS, não é o mais importante agora mas é algo barato de ajustar

```suggestion
          <h2>Cobranças e Pagamentos</h4>
```